torch/csrc/distributed/rpc/py_rref.cpp

```diff
@@ -95,7 +95,10 @@ TypePtr tryInferTypeWithTypeHint(
 
 ///////////////////////////  PyRRef  //////////////////////////////////
 
-PyRRef::PyRRef(c10::intrusive_ptr<RRef> rref) : rref_(std::move(rref)) {
+PyRRef::PyRRef(
+    c10::intrusive_ptr<RRef> rref,
+    const std::shared_ptr<FutureMessage> fm)
```

Comment:
Hmm, not sure how to fix the clang-tidy failure on this line, as I'm pretty sure I need to copy by value here to keep the future alive in some cases.